boards/arm/b_g474e_dpow1/board.c

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
this code seems to belong to a sample, not board level.

Reply:
This code was placed at the board level because the ADC reading of VBUS and USB-C configuration could very between devices. Although for these two samples, the code is similar.